lib/pf9-builder/testcase-02-ubuntu.csv

```diff
@@ -0,0 +1,6 @@
+## kubernetes
```

Comment:
Please remove these files from the pull request, we should ultimately distribute internally and add to the gitignore